sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Demo/Traces/TraceDemo.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using Azure.Monitor.OpenTelemetry.Exporter.Tracing.Customization;
+using OpenTelemetry;
+using OpenTelemetry.Extensions.AzureMonitor;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Demo.Traces
+{
+    internal class TraceDemo : IDisposable
+    {
+        private const string activitySourceName = "MyCompany.MyProduct.MyLibrary";
+        private static readonly ActivitySource activitySource = new(activitySourceName);
+
+        private readonly TracerProvider tracerProvider;
+
+        public TraceDemo(string connectionString)
+        {
+            var resourceAttributes = new Dictionary<string, object>
+            {
+                { "service.name", "my-service" },
+                { "service.namespace", "my-namespace" },
+                { "service.instance.id", "my-instance" },
+            };
+
+            var resourceBuilder = ResourceBuilder.CreateDefault().AddAttributes(resourceAttributes);
+
+            this.tracerProvider = Sdk.CreateTracerProviderBuilder()
```

Comment:
Its confusing that the ActivityEnrichingProcessor() and ActivityFilteringProcessor() are part of this project and not the SDK. Given that there are links to just this file as a sample, I got lost trying to find these - especially as github search is somewhat broken these days,

Reply:
HI @samsp-msft, point taken. I'll bring this up with the team about making these more discoverable. Thanks!